lib/db_bolt.go

```diff
@@ -0,0 +1,237 @@
+package lib
```

Comment:
@AeonSw4n  You can deprioritize the BoltDB exploration. We don't intend to merge this code for now anyway. You can move the code to a separate experimental branch.

For context, Nina & Zordon have been separately looking at reducing Badger memory usage by tuning the badger config options. Let's see where that ends up, and what the holistic memory usage of the final PoS node is before introducing a new KV data store.

Reply:
Moved to #617